### DISCUSSION TITLE
Uploading of the documents

### DISCUSSION THREAD

**Discussion by Javier505** [community/community#47394] [GitHub Education]
To whom it may concern,

I am trying to obtain the student package, but I can not upload the document certifying that I am an enroled student. The system only lets me take a picture, but I have a pdf.

Many thanks,
Javier.

**Comment by SilasRodrigues19** [comment]
Hey @Javier505 

Take a screenshot of the pdf

**Comment by Ntshangase** [comment]
you can convert that pdf into a JPG image that will work. To convert you can use ILOVEPDF website for free.

**Comment by Javier505** [comment]
Hello,

I am trying to upload it, but the only thing it lets me do is this:
[cid:0b060bc0-49a6-4acf-9ec0-3f78acd331d1]
________________________________
De: Mntwana ***@***.***>
Enviado: martes, 14 de febrero de 2023 18:15
Para: community/community ***@***.***>
Cc: Javier García Bravo ***@***.***>; Mention ***@***.***>
Asunto: Re: [community/community] Uploading of the documents (Discussion #47394)


CAUTION: [External Message] - Please be cautious when opening links or attachments in email. Make sure that the source and the content is safe.


you can convert that pdf into a JPG image that will work. To convert you can use ILOVEPDF website for free.

—
Reply to this email directly, view it on GitHub<https://eur02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Forgs%2Fcommunity%2Fdiscussions%2F47394%23discussioncomment-4973384&data=05%7C01%7Cjavier.gbravo%40alumno.ucjc.edu%7C1908ea4baab74d643b8a08db0eaf1ca1%7Cf962b9f9e2d949eeb98c14b402f6a5df%7C0%7C0%7C638119917492121456%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=%2F8fcC1yZDnYSA4E6IPj4o8fU57uI%2FdLg3A5ZjmG9474%3D&reserved=0>, or unsubscribe<https://eur02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FA54RTISDPZQIYDLWDM6KYBLWXO4UDANCNFSM6AAAAAAU3UCGSE&data=05%7C01%7Cjavier.gbravo%40alumno.ucjc.edu%7C1908ea4baab74d643b8a08db0eaf1ca1%7Cf962b9f9e2d949eeb98c14b402f6a5df%7C0%7C0%7C638119917492277679%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=BY4SSe5EcsZhky1vZ32WisC%2BSYGiDRVFZTsYU7L%2BYLk%3D&reserved=0>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

Aviso Legal: La información de este email es confidencial y exclusiva para la persona o entidad a la que va dirigido. Este email y/o cualquier documento adjunto pueden contener información de carácter privilegiado y/o estar protegidos de cualquier otra forma contra cualquier tipo de revelación. Si usted no es el destinatario de este email, por favor comunique este hecho al remitente de manera inmediata y borre el email y cualquier documento adjunto que pueda contener. No hay renuncia a la confidencialidad ni a ningún privilegio por causa de transmisión errónea o mal funcionamiento. Si usted no es el destinatario de este email queda advertido. No debe copiar el mensaje ni la información adjunta y que no debe revelar el contenido del mismo. © Institución Educativa SEK no es responsable de la correcta y completa transmisión de la información contenida en este email.



Disclaimer: The information contained in this email is confidential and is intended solely for the use of the individual or entity to whom it is addressed. This message and any attachment may contain trade secrets and may also be legally privileged or otherwise protected from disclosure. If you are not the intended recipient please notify the sender immediately and delete this message and any attachment form your system. No confidentiality nor any privilege regarding the information is waived or lost by any mistransmission or malfunction. If you are not the intended recipient you are on notice of its status therefore, please understand that you must not disseminate, distribute, copy or disclose the contents to any other person. © Institución Educativa SEK is neither liable for the proper and complete transmission of the information contained in this communication.

**Comment by samus-aran** [comment]
Hi @Javier505 👋 

We're happy you're here!

It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
2. If you can, print off your evidence and take a photo of this copy.
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

Let us know here if you need further assistance
